readthedocs/rtd_tests/tests/test_core_utils.py

```diff
@@ -33,6 +34,28 @@ def test_trigger_skipped_project(self, update_docs_task):
         self.assertEqual(result, (None, None))
         self.assertFalse(update_docs_task.signature.called)
         self.assertFalse(update_docs_task.signature().apply_async.called)
+    
+    @mock.patch('readthedocs.projects.tasks.update_docs_task')
+    def test_trigger_build_when_version_not_provided(self, update_docs_task):
+
+        trigger_build(project=self.project)
+        default_version = self.project.get_default_version()
```

Comment:
We should test this more explicitly, like `self.project.default_version = "someting"` and another case without `default_version`.